libraries/abstractions/wifi/test/iot_test_wifi.c

```diff
@@ -1134,7 +1171,22 @@ TEST( Full_WiFi, AFQP_WIFI_Scan_NullParameters )
     if( TEST_PROTECT() )
     {
         xWiFiStatus = WIFI_Scan( NULL, testwifiMAX_SCAN_NUMBER );
-        TEST_WIFI_ASSERT_NOT_EQ_REQUIRED_API( eWiFiSuccess, xWiFiStatus, xWiFiStatus );
+        TEST_WIFI_ASSERT_REQUIRED_API( xWiFiStatus != eWiFiSuccess, xWiFiStatus );
+    }
+}
+
+/**
+ * @brief Call WIFI_Scan() to scan for zero total networks and verify stability.
+ */
+TEST( Full_WiFi, AFQP_WIFI_Scan_ZeroScanNumber )
```

Comment:
These test cases and some of the others were removed from PR:
https://github.com/aws/amazon-freertos/pull/1685/files

Can you remove them from this PR.

Reply:
Got it. Removed in most recent local commit. Pushing shortly, a long with other review based changes